### PR TITLE
Enforce latests syntax on CI

### PR DESCRIPTION
**Pull request by mik-laj**
Continuation of: https://github.com/p1c2u/openapi-core/pull/344

### CODE REVIEW

File: openapi_core/templating/media_types/exceptions.py [hunk 1]
```diff
@@ -16,6 +16,6 @@ class MediaTypeNotFound(MediaTypeFinderError):
 
     def __str__(self):
         return (
-            "Content for the following mimetype not found: {0}. "
-            "Valid mimetypes: {1}"
-        ).format(self.mimetype, self.availableMimetypes)
+            f"Content for the following mimetype not found: {self.mimetype}. "
+            f"Valid mimetypes: {self.availableMimetypes}"
+        )
```
Comment (p1c2u) [inline openapi_core/templating/media_types/exceptions.py:21]: Add newline at the end of file

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+repos:
+  - repo: meta
+    hooks:
+      - id: check-hooks-apply
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.19.0
+    hooks:
+      - id: pyupgrade
+        args: ["--py36-plus"]
+        exclude: ^airflow/_vendor/
```
Comment (mik-laj) [inline .pre-commit-config.yaml:16]:
```suggestion
```

File: openapi_core/templating/media_types/exceptions.py (hunk 1 above)
Comment (mik-laj) [inline openapi_core/templating/media_types/exceptions.py:21]:
```suggestion
        )

```

**Comment by mik-laj** [comment]
@p1c2u  All done

**Review by p1c2u — APPROVED** [review]
LGTM

**Comment by p1c2u** [comment]
@mik-laj thanks for the contribution

**Pull request merged**